### ISSUE TITLE
Packaging request: Fragments

### ISSUE THREAD

**Issue by onny** [NixOS/nixpkgs#132316]
**Project description**
Fragments is a modern, responsive and graphical Gnome bittorrent client

**Metadata**

* homepage URL: https://gitlab.gnome.org/World/Fragments
* source URL: https://gitlab.gnome.org/World/Fragments
* license: gpl3
* platforms: linux

The new version of Fragments depends on libutp which hopefully gets merged soon. I've opened a pull request for it [here](https://github.com/NixOS/nixpkgs/pull/132315).

[Earlier closed pull request](https://github.com/NixOS/nixpkgs/pull/56793) for Fragments.

**Comment by davidak** [comment]
We have transmission and torrential. Maybe that's something for you.

**Comment by onny** [comment]
That's true but Fragments is a really neat, easy and modern looking torrent client. It has an adaptive interface which would also work quite well on mobile devices, for example NixOS mobile :+1:
